etc/scripts/find-node

```diff
@@ -24,7 +24,7 @@ find_node() {
 
 NODE=$(find_node)
 
-NODE_MIN_VERSION=8
```

Comment:
Update https://github.com/mattgodbolt/compiler-explorer/blob/master/CONTRIBUTING.md#node-version to reflect this change

Reply:
Done: thanks!